Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/submissionDetailsService.ts

```diff
@@ -29,9 +32,15 @@ const submissionDetailsService = createApi({
                 const blob = await response.blob();
                 return { blob, filename };
             }
+
+            if (response.headers.get('Content-Length')) {
+                return '';
+            }
```

Comment:
What made this necessary?

Reply:
Don't really remember anymore but there was an issue in redux.